bin/build_docs.py

```diff
@@ -62,6 +62,23 @@ def main():
         sphinx_apidoc_cmd + ["sphinx_gherkindoc", "*sample-conf*"], cwd=BASE_DIR
     )
 
+    print("Building Sample Gherkin docs")
+    subprocess.check_call(
+        [
+            "poetry",
+            "run",
+            "sphinx-gherkindoc",
+            "--doc-project",
+            "Sample Gherkin",
+            "--toc-name",
+            "sample-gherkindoc",
+            "--step-glossary-name",
+            "sample-gherkindoc-glossary",
+            "tests",
+            DOCS_WORKING_DIRECTORY,
+        ]
+    )
```

Comment:
I think that for #11 at least, I'd end up making another example like this to show what it looks like with the background integrated.

Reply:
I think that would be a good idea. I'm a big fan of "Ok, yeah, I read how this works, but _show me_ what it looks like." So if we have multiple options like that, we should definitely show them off!